app-tasks/rf/src/rf/ingest/io.py

```diff
@@ -0,0 +1,145 @@
+"""Utilities for transforming public scenes into COGs"""
+
+from rf.ingest.settings import (landsat8_band_order, sentinel2_band_order,
+                                landsat8_datasource_id,
+                                sentinel2_datasource_id)
+from rf.uploads.landsat8.io import get_tempdir
+from rf.utils.io import s3_bucket_and_key_from_url
+
+import boto3
+
+import logging
+from multiprocessing import (cpu_count, Pool)
+import os
+import subprocess
+
+DATA_BUCKET = os.getenv('DATA_BUCKET')
+
+s3client = boto3.client('s3')
+logger = logging.getLogger(__name__)
+
+
+def create_cog(image_locations, scene):
+    with get_tempdir() as local_dir:
+        dsts = [os.path.join(local_dir, fname) for _, fname in image_locations]
+        fetch_imagery(image_locations, local_dir)
+        merged_tif = merge_tifs(dsts, local_dir)
+        add_overviews(merged_tif)
+        cog_path = convert_to_cog(merged_tif, local_dir)
+        updated_scene = upload_tif(cog_path, scene)
+        updated_scene.update()
+
+
+def add_overviews(tif_path):
+    logger.info('Adding overviews to %s', tif_path)
+    overviews_command = ['gdaladdo', tif_path]
```

Comment:
So you can actually add some options here and I think it would be good.

` --config COMPRESS_OVERVIEW=DEFLATE --config INTERLEAVE_OVERVIEW=BAND`

Not sure if there are others, but those are the ones I noticed from the help docs

Reply:
I put `INTERLEAVE=BAND` in the cog creation commands elsewhere, does it need to be here instead?

Reply:
I think probably in both spots would be good, but I have no evidence

Reply:
I didn't even find it anywhere on the `gdaladdo` page -- I don't think it does anything there.
Since it's in "Creation Options" [here](https://www.gdal.org/frmt_gtiff.html), I think putting it behind a `-co` in the `gdal_merge` should do it. I'm gonna try putting it there.